masterlist.yaml

```
@@ -21763,6 +21763,34 @@ plugins:
       - crc: 0x1BE50A99
         util: 'SSEEdit v4.0.4'
 
+  - name: 'RynsWhiterunCityLimits.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65661/'
+        name: 'Ryn''s Whiterun City Limits'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'eFPS - Exterior FPS boost'
+          - '[Unofficial eFPS patches](https://www.nexusmods.com/skyrimspecialedition/mods/70176/)'
+        condition: 'active("Occ_Skyrim_Tamriel.esp") and not active("Occ_Skyrim_RynsWhiterunCityLimits_patch")'
```

Missing file extension.
```suggestion
        condition: 'active("Occ_Skyrim_Tamriel.esp") and not active("Occ_Skyrim_RynsWhiterunCityLimits_patch.esp")'
```